test/functional/tests/security/fuzzy/kernel/common/common.py

```diff
@@ -22,43 +24,58 @@ def get_fuzz_config(config_name: str):
     return fuzz_config
 
 
-def prepare_cas_instance(cache_disk, core_disk, cache_mode: CacheMode = None,
+def prepare_cas_instance(cache_device, core_device, cache_mode: CacheMode = None,
                          cache_line_size: CacheLineSize = None,
                          kernel_params: KernelParameters = KernelParameters(),
-                         cleaning_policy: CleaningPolicy = None, mount_point: str = None):
+                         cleaning_policy: CleaningPolicy = None, mount_point: str = None,
+                         create_partition=True):
     #  Change cleaning policy to default for Write Policy different than WB
     if cleaning_policy:
         cleaning_policy = CleaningPolicy.DEFAULT if cache_mode != CacheMode.WB \
             else cleaning_policy
 
-    cache_disk.create_partitions([Size(400, Unit.MebiByte)])
-    cache_device = cache_disk.partitions[0]
+    if create_partition is True:
+        cache_device.create_partitions([Size(400, Unit.MebiByte)])
+        cache_device = cache_device.partitions[0]
+
     cache = casadm.start_cache(cache_device, cache_mode, cache_line_size, 1, True,
                                kernel_params=kernel_params)
     if cleaning_policy:
         cache.set_cleaning_policy(cleaning_policy)
 
     if mount_point:
-        core_disk.create_filesystem(Filesystem.ext4)
-        core = cache.add_core(core_disk)
+        core_device.create_filesystem(Filesystem.ext4)
+        core = cache.add_core(core_device)
         core.mount(mount_point)
     else:
-        core = cache.add_core(core_disk)
+        core = cache.add_core(core_device)
 
     return cache, core
 
 
-def run_cmd_and_validate(cmd, value_name: str, valid_values: list,
+def run_cmd_and_validate(cmd, value_name: str, is_valid: bool,
```

Comment:
After this change `post_process_param_func` parameter won't be useful anymore

Reply:
There are different tests that use `valid_values` param and removing it is going to break these tests. If you want this change to be merged, you need to update those tests